viam_cartographer.go

```diff
@@ -32,6 +32,8 @@ var (
 	cartoLib cartofacade.CartoLib
 	// ErrClosed denotes that the slam service method was called on a closed slam resource.
 	ErrClosed = errors.Errorf("resource (%s) is closed", Model.String())
+	// ErrRunSlamDisabled denotes that the slam service method was called while run_slam was set to false
+	ErrRunSlamDisabled = errors.Errorf("resource (%s) was booted with run_slam set to false", Model.String())
```

Comment:
@JohnN193 how does this error messages look?

Reply:
how does `"resource (%s) unavailable, configured with run_slam set to false"` sound? the initial message isn't clear to me why I am seeing the error. 

Other part I am thinking about is if we should say something about cloud slam, so users don't try to use the endpoints while cloud slam is running

Reply:
updated. I'm not really sure how to describe in a single line error message anything about cloud slam but am happy to integrate any requests for error message changes.

Reply:
same tbh, but fine to leave as is